docs/source/overview/capabilities.rst

```diff
@@ -29,11 +15,51 @@ server, lambda).
 When a user runs Custodian, Custodian will run the specified policy against the account
 and region specified by the user. Custodian will iterate over all resources
 defined in the policy. In the CLI, users specify the account and region they want
-to target. During the run, each policy in the config will generate metrics that
-are sent to CloudWatch in the account that is targeted. The run will also
-generate structured record output and logs that can be sent to an S3 Bucket and
-CloudWatch Log Groups in the account Custodian was run from. If Custodian is being
-run without Assume Roles, all output will be put into the same account. Custodian
-is built with the ability to be run from different accounts and leverage STS
+to target.
+
+During the run, each policy in the config will generate metrics that can be sent to
+the cloud provider's built-in metrics service (CloudWatch, Application Insights, Stackdriver)
+in the account or subscription that is targeted. The run will also generate structured record
+output and logs that can be sent to the cloud provider's blob storage service (S3,
+Azure Storage Accounts, Google Cloud Storage Bucket) or logging service (CloudWatch
+Logs, Azure Application Insights Logs, Stackdriver logs) in the account Custodian was
+run from.
+
+Custodian currently provides policy definition around AWS, Azure, and Google Cloud
+Platform resources:
+
+
+AWS
+---
+
```

Comment:
this needs a note that this list is partial, and to reference the full list at the moment (till its fully in the docs) they should be look at `custodian schema aws` output.